src/resources/lib/modules/connman.py

```diff
@@ -12,7 +12,7 @@
 import xbmcgui
 import threading
 import oeWindows
-import ConfigParser
+import configparser
```

Comment:
Where is `configparser` being used? Looks like we can do this `import`?

Reply:
s/do/drop/

Reply:
I found the reference all the way back OpenELEC 5 - under the load_vpn_config function.

I don't believe this module is needed - I would say we can test dropping it.

Reply:
Yeah, should have been removed as part of this commit:

https://github.com/OpenELEC/service.openelec.settings/commit/5a97990b6fa87b8a09ac7a4ce5d4ec62778340b5